SpatialGDKEditorToolbar/Source/Private/SpatialGDKEditorGenerateSnapshot.cpp

```diff
@@ -48,12 +48,12 @@ bool CreateSpawnerEntity(Worker_SnapshotOutputStream* OutputStream)
 	TArray<Worker_ComponentData> Components;
 
 	WriteAclMap ComponentWriteAcl;
-	ComponentWriteAcl.Add(POSITION_COMPONENT_ID, UnrealWorkerPermission);
-	ComponentWriteAcl.Add(METADATA_COMPONENT_ID, UnrealWorkerPermission);
-	ComponentWriteAcl.Add(PERSISTENCE_COMPONENT_ID, UnrealWorkerPermission);
-	ComponentWriteAcl.Add(UNREAL_METADATA_COMPONENT_ID, UnrealWorkerPermission);
-	ComponentWriteAcl.Add(ENTITY_ACL_COMPONENT_ID, UnrealWorkerPermission);
-	ComponentWriteAcl.Add(SpatialConstants::PLAYER_SPAWNER_COMPONENT_ID, UnrealWorkerPermission);
+	ComponentWriteAcl.Add(SpatialConstants::POSITION_COMPONENT_ID, UnrealServerPermission);
```

Comment:
Could the creation of a ComponentWriteAcl which sets up permissions for all the standard components be put into a helper? I think `SpatialSender` could also use it.

Reply:
Yep good idea. I'll do it in a separate PR though, to get this merged now.

Reply:
Do we have a ticket?

Reply:
UNR-585